prime-angular-frontend/src/app/modules/adjudication/pages/health-authorities/health-auth-org-info-page/health-auth-org-info-page.component.ts

```diff
@@ -24,15 +26,20 @@ export class HealthAuthOrgInfoPageComponent implements OnInit {
    * has not yet entered their organization information.
    */
   public isInitial: boolean;
+  public upload: boolean;
+
+  public orgAgreeDocument: BaseDocument;
 
   private routeUtils: RouteUtils;
 
   constructor(
     private healthAuthResource: HealthAuthorityResource,
     private route: ActivatedRoute,
-    router: Router
+    router: Router,
```

Comment:
Although it was this way prior to this PR, can `private` be added here?

Reply:
Had to ask Marty again the reasoning, but we do not indicate private/public on things we inject and only want to use inside the constructor. So this way you cannot use router outside the initialization of the routing util. Since we should be using the routeUtils to handle our routing.